### PR TITLE
Install and configure webpack with babel

### PR DESCRIPTION
**Pull request by ushakouk**
Install and configure webpack with babel,
Create dev and prod build scripts,
Use arguments instead of variables for separate build config.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -3,36 +3,26 @@
...
-    "test": "react-scripts test",
-    "eject": "react-scripts eject"
-  },
-  "eslintConfig": {
-    "extends": [
-      "react-app",
-      "react-app/jest"
-    ]
+  "devDependencies": {
+    "babel-core": "^6.26.3",
```
Comment (NikitaLens) [inline package.json:10]: delete all ^

File: webpack.config.js [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+  entry: './src/index.js',
+
+  output: {
+    filename: '[name].bundle.js',
+    path: path.join(__dirname, 'built'),
+  },
+
+  resolve: {
+    modules: [path.join(__dirname, './src'), 'node_modules'],
+    extensions: ['.js']
```
Comment (NikitaLens) [inline webpack.config.js:20]: add .jsx

File: package.json (hunk 1 above)
Comment (ushakouk) [reply package.json]: removed to avoid automatically updates

File: webpack.config.js (hunk 2 above)
Comment (ushakouk) [reply webpack.config.js]: Done with changed App component file extension to jsx

**Review by NikitaLens — APPROVED** [review]

**Pull request merged**